lib/ae_mdw_web/controllers/tx_controller.ex

```diff
@@ -58,7 +61,32 @@ defmodule AeMdwWeb.TxController do
     end
   end
 
-  def read_tx_hash(tx_hash) do
+  #
+  # Private functions
+  #
+  defp add_spendtx_details(data_txs, %{"account" => account}) do
+    Enum.map(data_txs, fn block ->
+      spend_tx_recipient =
+        if block["tx"]["type"] == @type_spend_tx, do: block["tx"]["recipient_id"]
+
+      if nil != spend_tx_recipient and String.slice(spend_tx_recipient, 0..2) == "nm_" do
+        case Validate.plain_name(spend_tx_recipient) do
+          {:ok, name} ->
+            Map.merge(block, %{"name" => name, "account" => account})
+
+          _ ->
```

Comment:
I think unused variables are more useful to the reader if we explicitly name them what they are supposed to be (e.g. `{:error, _reason}` here and `add_spendtx_details(data_txs, _params)` down below.